SeaPublicWebsite/Views/EnergyEfficiency/Recommendations/InsulateYourLoft.cshtml

```diff
@@ -0,0 +1,65 @@
+﻿@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model SeaPublicWebsite.Models.EnergyEfficiency.RecommendationViewModel;
+
+@{
+    Layout = "_Recommendation";
+}
+
+@section RecommendationBody
+{
+    <p class="govuk-body">
+        Based on the information we have collected; it looks like you have no insulation in your loft. 
+        We recommend that you have your loft insulated, if possible, 
+        up to the recommended thickness of 300mm, or 12 inches.
+    </p>
+
+    <p class="govuk-body">
+        Adding insulation will help trap the heat inside your property so your heating doesn’t need to be on as much, 
+        and so could make a significant saving on you fuel bills.
```

Comment:
Typo - "you fuel bills" should be "your fuel bills"

Where did this copy originate from?

Reply:
The copy was written during the private beta, in a sharepoint document, but it was out of scope to add it then